test/integration/scheduler/plugins/plugins_test.go

```diff
@@ -2686,7 +2687,7 @@ func TestSchedulingGatesPluginEventsToRegister(t *testing.T) {
 		{
 			name:              "preEnqueue plugin with event registered",
 			withEvents:        true,
-			count:             2,
+			count:             3,
```

Comment:
This is a correct change -
This test uses a custom PreEnqueue plugin, and thus this PR makes the test case not go through a shortcut.
Actually before https://github.com/kubernetes/kubernetes/pull/124618, the value here was 3.

Reply:
Let's merge this first https://github.com/kubernetes/kubernetes/pull/124783, no need to worry about this anymore. Can you take a look.

Reply:
OK, I'll review that one though, can we regard this PR as independent from that one?
I don't think it's necessary to block this PR waiting for your PR. This PR is high-prioritized to resolve master-blocking job failure https://github.com/kubernetes/kubernetes/issues/125223.

Reply:
I'm ok to merge this first, just let you know you don't have to focus on this. 

Maybe we should merge this first if we want to cherry pick.

Reply:
Actually, after taking a look at https://github.com/kubernetes/kubernetes/pull/124783, I now understand your PR wouldn't take so much time to merge :) Let's see anyway.

Reply:
#124783 should not be cherry-picked, to avoid any unforeseen problems.

Now this change is very confusing, because the test and comment refer specifically to SchedulingGates.
Let's change those.

Reply:
@alculquicondor 
Sorry, so what are you expecting me to change? If we don't cherry-pick #124783, we have to change this number to make the test case pass, right?

Reply:
yes, change the number, but also change the name of the test and corresponding comment.

Reply:
Now I got it, updated the test name and the comment on top of that.